code/utility/src/palindrome/palindrome_check/palindrome_check.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <string.h>
+
+#define MAX_STRING_LEN 256
+
+void isPalindrome(char str[]) //Iterative function to check if a string is palindrome
```

Comment:
Make this function a boolean function that returns true or false. Functions should have limited responsibilities. Don't put any output statements in the function. Leave that to main.